projects/documentation/package.json

```
@@ -239,7 +238,7 @@
             ],
             "dependencies": [
                 {
-                    "script": "../../:docs:analyze",
+                    "script": "../../:docs:cem",
```

```suggestion
                    "script": "../../:build:cem:docs",
```

I believe this is the name of the new command